### DISCUSSION TITLE
CONNECTING TO WALLET

### DISCUSSION THREAD

**Discussion by Cyfren** [Chia-Network/chia-blockchain#5085] [Support]
HI
i have an issue with chia because i have 2 days and are not starting again after i restart the pc 
Always are showing me Connecting to wallet and never start 
i try do delete and reinstall but always i get same error
Can someone help me with this please?
![Screenshot_1](https://user-images.githubusercontent.com/79423266/118354210-3ed15880-b56a-11eb-9c71-7546871d476d.png)

**Comment by FSys77** [comment]
imho Chia software is a piece of shit. It is unstable, flaky and hacky as hell. When I compare it to the first versions of the Bitcon software they are worlds apart. Bitcoin simply worked, I never had issues with clients not starting or sync errors.

Chia software is still in beta - or even alpha.

**Comment by Helheimbaby** [comment]
Got the same Problem here.

System run over a Week without any issues then after restart same "Connecting to Wallet" like Cyfren has.

**Comment by DylannJ** [comment]
I delete the folder .chia locate in C:\user\yourname\ .chia and its work but chia restart to zero, I have all my plot but the sync restart to zero

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
